src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -48,117 +48,121 @@ export default function SearchBar({
   };
 
   return (
-    <div className="vads-u-border-bottom--0 medium-screen:vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-top--3 vads-u-padding-bottom--0 medium-screen:vads-u-padding-bottom--3 vads-u-padding-x--1">
-      {/* Mobile expand/collapse */}
-      <button
-        className={`${
-          expanded ? 'va-border-bottom-radius--0 ' : ''
-        }vads-u-width--full vads-u-display--flex vads-u-margin--0 vads-u-padding-x--2 vads-u-justify-content--space-between vads-u-padding-y--2 vads-u-color--primary-darker vads-u-background-color--gray-lightest medium-screen:vads-u-display--none`}
-        onClick={() => setExpanded(!expanded)}
-        type="button"
-      >
-        Search resources and support
-        <i
+    <div className="vads-u-border-bottom--0 medium-screen:vads-u-border-top--2px vads-u-border-color--gray-light vads-u-padding-top--3 vads-u-padding-bottom--0 medium-screen:vads-u-padding-bottom--3">
+      <div className="vads-u-padding-x--1 large-screen:vads-u-padding-x--0">
```

Comment:
Just added the `<div className="vads-u-padding-x--1 large-screen:vads-u-padding-x--0">` wrapper. See diff w/o whitespace for a better diff